chef/cookbooks/trove/recipes/default.rb

```diff
@@ -98,7 +98,7 @@ class ::Chef::Recipe
 # rubygem-mysql is installed here, although it would normally be the
 # database cookbook's responsibility. The database cookbook uses a
 # special mysql-chef_gem for this which is Chef 0.11 only.
-pkgs = ['mysql', 'python-mysql']
+pkgs = ['mariadb', 'python-mysql']
```

Comment:
MariaDB is in the meanwhile standard in the most Linux distros. So I see there no need for this.